packages/runtime/container-runtime/src/summaryCollection.ts

```diff
@@ -205,14 +207,18 @@ export class SummaryCollection {
     private lastSummaryTimestamp: number | undefined;
     private maxAckWaitTime: number | undefined;
     private pendingAckTimerTimeoutCallback: (() => void) | undefined;
-    private lastAck?: IAckedSummary;
+    private lastAck: IAckedSummary | undefined;
 
     public get latestAck() { return this.lastAck; }
 
     public constructor(
-        public readonly initialSequenceNumber: number,
+        private readonly deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>,
         private readonly logger: ITelemetryLogger,
-    ) { }
+    ) {
+        this.deltaManager.on(
+            "op",
+            (op) => this.handleOp(op));
```

Comment:
this is a minor change. before it would listen on `incoming.on("op"` however this will miss ops not coming from the delta stream aka delta storage, but we want the collection to see all summary attempts, even those that are old.